rust/feature-flags/src/cohort/cohort_cache_manager.rs

```diff
@@ -70,11 +70,7 @@ impl CohortCacheManager {
     pub async fn get_cohorts(&self, project_id: ProjectId) -> Result<Vec<Cohort>, FlagError> {
         // First check cache before acquiring lock
         if let Some(cached_cohorts) = self.cache.get(&project_id).await {
-            common_metrics::inc(
-                COHORT_CACHE_HIT_COUNTER,
-                &[("project_id".to_string(), project_id.to_string())],
-                1,
-            );
```

Comment:
`project_id` (along with `team_id`) is going to be too high cardinality for prometheus to handle, so it's not worth labeling the metrics with this level of granularity.